components/ModEditFacilityOrProfessionalTopbar.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+    <div class="flex flex-row justify-between w-full">
+        <div>
+            <button
+                data-testid="mod-edit-facility-hp-topbar-copy-id"
+                class="flex flex-row w-90 bg-neutral p-2 m-2 border-2 border-slate-400 rounded hover"
+                @click="copyFacilityOrHPId"
+            >
+                ID: {{ selectedId }}
+                <SVGSuccessCheckMark
+                    v-if="showCopySuccessIcon"
+                    role="img"
+                    title="clipboard copy"
+                    class="ml-2 w-6"
+                />
+                <SVGCopyContent
+                    v-else
```

Comment:
personal pref: I think using two v-ifs is easier to read and less risky than `v-else`.

Reply:
I also prefer if statements over defaulting to `elses`